pages/merchant/_reducer.tsx

```diff
@@ -0,0 +1,103 @@
+import router from "next/router"
+import React from "react"
+
+export const ACTIONS = {
+  ADD_DIGIT: "add-digit",
+  DELETE_DIGIT: "delete-digit",
+  CLEAR_INPUT: "clear-input",
+  CREATE_INVOICE: "create-invoice",
+  CREATE_NEW_INVOICE: "create-new-invoice",
+  GET_PAYMENT_STATUS: "get-payment-status",
+  BACK: "back-by-one-history",
+}
+
+export type ACTIONTYPE = { type: string; payload?: string | (() => void) }
```

Comment:
Not a big deal - but shouldn't this be `ACTION_TYPE`?

Reply:
I'll change that